packages/ui/cart/drawer/CartLineItem.tsx

```diff
@@ -68,19 +69,70 @@ export function CartLineItem({ lineItem }: CartLineItemProps) {
          itemcode: lineItem.itemcode,
          quantity: 1,
       });
+      const shopifyVariantDecoded = Buffer.from(
+         lineItem.shopifyVariantId,
+         'base64'
+      );
```

Comment:
`The signature '(data: string): string' of 'atob' is deprecated.ts(6387)`

Reply:
`Buffer` is only defined in Node, I think this will throw a ReferenceError in the browser

Reply:
Hm this passed QA though...

Reply:
Shopify has used two different formats for their ids. Sometimes they'd use an int, and sometimes they'd use a base64 encoded global id, which is what we're doing here. I think their guidance on which to use has changed though. Rather than jump through the hoop of converting one to the other on the client, it might make sense to add the other format to our api (shopifyVariantIdInt or something) or possibly switch entirely and standardize on one format if possible. Though that might also be too much for this issue.

Reply:
i think sorting that all out would be a good idea @sterlinghirsh, but i think its out of scope of this issue like you said